index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Anonymous+Pro&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <a href="#" class="link">Jesse McCallums</a>
+        <nav>
+            <i class="bi bi-list"></i>
+        </nav>
+    </header>
+
+    <main>
+        <section>
+            <h1 class="firsttext">Jesse McCallums</h1>
+            <h2 class="secondtext"> Software developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+    </main>
```

Comment:
- [ ] Kindly add all the icons, buttons, and content present in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10). I can see that your design is very similar to Figma template 4 but there is more content in this section of the template. Please, stick to the template design as much as possible.
- [ ] Also make sure you are using the background images as in the template.

![image](https://user-images.githubusercontent.com/61919617/170545374-f1498332-d149-400e-b399-dda4a2288a80.png)
![image](https://user-images.githubusercontent.com/61919617/170545459-1207bfd5-7dd2-43d0-bccf-db3e7039022d.png)